contracts/tokens.md

```diff
@@ -2,13 +2,53 @@
 
 ## gOHM
 
-gOHM stands for Governance OHM. It supersedes [wsOHM](#wsOHM) as part of the [v2
-migration](../basics/migration.md).
+gOHM stands for Governance OHM. It supersedes [wsOHM](#wsohm) as part of the [v2
+migration](../basics/migration.md). gOHM is wrapped sOHM V2, which allows you to
+use sOHM V2 on different blockchains. It is priced exactly the same as wsOHM:
+
+$$
+gOHM_{price} = OHM_{price} * CurrentIndex
+$$
+
+You still collect rebase rewards just as if you had sOHM, but you won't see your
+token balance increase because the increase in value is based on the Current Index
+at the time of purchase and sale. [See this FAQ](../basics/basics.md#how-do-i-track-my-rebase-rewards)
+for more details.
+
 Below are listed gOHM contracts by version, where the latest version represents
 the currently active contract.
 
+ETH Mainnet:
+
 * V1 [0x0ab8...a52f](https://etherscan.io/address/0x0ab87046fBb341D058F17CBC4c1133F25a20a52f)
 
+Arbitrum One:
+
+* V1 [0x8D9b...5FB1](https://arbiscan.io/token/0x8D9bA570D6cb60C7e3e0F31343Efe75AB8E65FB1)
+
+AVAX Chain:
```

Comment:
I would maybe call it like this. Just not sure right now if there has to be a more specific separation to be made. Like there is the C-Chain and whatnot. 

```suggestion
Avalanche:
```